### DISCUSSION TITLE
Discord

### DISCUSSION THREAD

**Discussion by ghostdevv** [Budibase/budibase#1719] [Help] [answered]
Hey all, was wondering if you had a Discord Server?

**Comment by joebudi** [comment, marked as answer]
Hey. Yes, we do, but we have asked our community to use Github Discussions for anything feature, support, general chit chat related.

Here's the Discord link:
https://discord.gg/r9CnWuA4

But please take note of the above and use Github Discussions, if possible.

**Reply by ghostdevv** [reply]
Thank you, no worries will use discussions :smile:

**Reply by brocococonut** [reply]
Was there a reason the Discord was discontinued? I generally find it easier to get faster support and feedback from community members via Discord
